emulators/mednaffe.md

```diff
@@ -0,0 +1,72 @@
+# Mednaffe (Mednafen) guide
+
+ ❗ Follow the [Getting Started section](../README.md#getting-started) of `README.md` first.
+
+This section of the guide explains how to setup [Mednaffe](https://github.com/AmatCoder/mednaffe), a front-end for [Mednafen](https://mednafen.github.io/), a multi-system emulator.
+
+<img src="https://user-images.githubusercontent.com/9209426/157373278-adeecff8-726d-414d-a599-5ac4fa143afb.png" title="" alt="A screenshot of Snes9x running Super Mario World" data-align="inline">
+
+## Installing Mednaffe
+
+Open Discover, SteamOS' app store, then search for **Mednaffe**, then click on it.
+
+![Discover](https://user-images.githubusercontent.com/9209426/157377104-dfa105bb-d82c-401a-bb7d-d5590585c2cc.png)
+
+In the top right, select **Sources**, then **Flatpak**, then press **Install**.
+
+Alternatively, open up a terminal and run
+
+```
+flatpak install --user -y com.github.AmatCoder.mednaffe
+```
+
+## Configuring Mednaffe
+
+### Issues with sound
+
+Open up the emulator, then navigate to **Global Settings > Sound**. Make sure `Enable sound output` is ticked, and that `Driver` is set to `sdl`, otherwise you won't get any sound. 
+
+### Input mapping
+
+Switch to the **Systems** tab and there, for each system, you have an **Input** sub-tab to map various controllers as you wish. Assuming you launched the emulator from within Desktop mode, you'll need to have Steam launched beforehand to map Steam Deck inputs properly (otherwise it won't be recognized as a controller).
```

Comment:
```suggestion
Switch to the **Systems** tab. For each system, you can map various controllers under the **Input** sub-tab. In Desktop mode, you'll need to launch Steam before mapping Steam Deck inputs properly, otherwise it won't be recognized as a controller.
```